tests/unittests/tests-cncbor/tests-cncbor.c

```diff
@@ -0,0 +1,191 @@
+﻿/*
+ * Copyright (C) Lorenz Hüther, Mathias Detmers
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ /**
+  * @ingroup    tests
+  * @{
+  *
+  * @file
+  * @brief      Unit tests for pkg cn-cbor.
+  *
+  * @author     Lorenz Hüther
```

Comment:
if possible, please add email addresses

Reply:
Fine with me, I've got to ask Mathias first though.